go/enclave/container/enclave_container.go

```diff
@@ -5,15 +5,15 @@ import (
 	"fmt"
 
 	gethlog "github.com/ethereum/go-ethereum/log"
-	"github.com/obscuronet/go-obscuro/go/common"
-	"github.com/obscuronet/go-obscuro/go/common/log"
-	"github.com/obscuronet/go-obscuro/go/enclave"
+	"github.com/ten-protocol/go-ten/go/common"
+	"github.com/ten-protocol/go-ten/go/common/log"
+	"github.com/ten-protocol/go-ten/go/enclave"
 
-	"github.com/obscuronet/go-obscuro/go/config"
+	"github.com/ten-protocol/go-ten/go/config"
 
-	"github.com/obscuronet/go-obscuro/go/ethadapter/mgmtcontractlib"
+	"github.com/ten-protocol/go-ten/go/ethadapter/mgmtcontractlib"
 
-	obscuroGenesis "github.com/obscuronet/go-obscuro/go/enclave/genesis"
+	obscuroGenesis "github.com/ten-protocol/go-ten/go/enclave/genesis"
```

Comment:
Guess changing aliases like these can come later as we update the code references?

Reply:
yeah. that's what I have in mind. 
a step by step approach, so it can be reviewed sanely